### ISSUE TITLE
User menu is not keyboard accessible

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons#10747]
The dropdown user menu needs to be keyboard accessible otherwise those links are completely obscured to anyone that can't use a mouse.

The username menu should be focusable, and when focused that should open the menu and make it possible to tab down through the links.

<img width="405" alt="add-ons_for_firefox" src="https://user-images.githubusercontent.com/1514/30370757-6789c12e-9870-11e7-9c66-f31dcb4151f4.png">
